app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -1,4 +1,4 @@
-$dropdown-height: 373px; // Allows for entire hero_simple text to display from $screen-md and up
+$dropdown-height: 325px; // Allows for entire hero_simple text to display from $screen-md and up
```

Comment:
This change falls down here.

![screen shot 2016-12-30 at 10 59 36 am](https://cloud.githubusercontent.com/assets/3814998/21568048/14e7ff6e-ce7f-11e6-8a0c-f76346d3b95f.png)